includes/data-stores/class-wc-admin-reports-products-data-store.php

```diff
@@ -177,7 +183,14 @@ protected function include_extended_info( &$products_data, $query_args ) {
 				$product             = wc_get_product( $product_data['product_id'] );
 				$extended_attributes = apply_filters( 'woocommerce_rest_reports_products_extended_attributes', $this->extended_attributes, $product_data );
 				foreach ( $extended_attributes as $extended_attribute ) {
-					$function = 'get_' . $extended_attribute;
+					if ( 'variations' === $extended_attribute ) {
```

Comment:
Thanks for the review @timmyc!

The problem is that `$extended_attribute`'s value is `'variations'` while the method we have to use is `get_children()`, so in this case `'get_' . $extended_attribute` wouldn't work.

For the attribute name, I used `'variations'` instead of `'children'` to make it clear we are only returning children for variable products. What do you think?